### DISCUSSION TITLE
Llama 2 language model feature

### DISCUSSION THREAD

**Discussion by devcpra** [chidiwilliams/buzz#552] [Ideas]
Since Facebook has now released Llama 2, will you be including it in your program as a seperate language model or even in conjunction to GPT?

**Comment by raivisdejus** [comment]
Latest versions of Buzz also support translations using AI. For usage see https://chidiwilliams.github.io/buzz/docs/usage/translations

Some custom APIs are specified here https://github.com/chidiwilliams/buzz/discussions/827

And t ospecify different APIs for transcription and translation you can use the advanced settings environment variable https://chidiwilliams.github.io/buzz/docs/preferences#advanced-preferences

**Comment by devcpra** [comment]
My actual question was related to transcription and not translation. I did not state that correctly. Currently, it is Whisper, Whisper.cpp and Faster Whisper models within the Buzz app (windows). Will there be a model of Llama 2 for transcription, or did I not uderstand what your reply was? My question is only related to transcription and not translation currently. 

![Llama 2](https://github.com/user-attachments/assets/bcb010c1-9e65-4a86-8d13-7d8efffaac39)

**Comment by raivisdejus** [comment]
Translations in "transcription" phase are designed only to translate to English. That is how Whisper models work. If you need to translate to any other language you need to use the AI translation feature.

Llama2 is not designed to recognize speech. There are other non-whisper models that can do it, but there are no concrete plans to introduce them in Buzz.

**Comment by devcpra** [comment]
Okay, got it. I was wondering if other models would be introduced or not.
